src/libs/asyncOpenURL/index.website.js

```diff
@@ -0,0 +1,29 @@
+import {Linking} from 'react-native';
+
+/**
+ * Prevents Safari from blocking pop-up window when opened within async call.
+ *
+ * @param {Promise} promise
+ */
+export default function asyncOpenURL(promise) {
+    const isSafari = /^((?!chrome|android).)*safari/i.test(navigator.userAgent);
```

Comment:
Can you explain why are you doing a negative patten match for chrome or android here? Is there some case when the user agent would be formatted like 'chrome-xxx-safari' or something?

Reply:
@roryabraham I think all third party browsers on iOS are just skins for safari.
So chrome on iOS has this issue too. And this pattern detects that in my testing :) 

The negative pattern match is to exclude both chrome and edge, and android.

Edge and Chrome's userAgent: `"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36"`

Reply:
Thanks for answering this @rushatgabhane! Yes, that is the reason I used the negative pattern match. @roryabraham Do we have any other preferred method?